Makefile.autosetup

```diff
@@ -360,6 +360,9 @@ LIBDEBUGOBJS+=	debug/parse_test.o
 @if USE_DEBUG_WINDOW
 LIBDEBUGOBJS+=	debug/window.o
 @endif
+@if USE_DEBUG_EMAIL || USE_DEBUG_GRAPHVIZ
+LIBDEBUGOBJS+=	debug/common.o
+@endif
 @if HAVE_LIBUNWIND || USE_DEBUG_COLOR || USE_DEBUG_EMAIL || USE_DEBUG_GRAPHVIZ || USE_DEBUG_NOTIFY || USE_DEBUG_PARSE_TEST || USE_DEBUG_WINDOW
 LIBDEBUG=	libdebug.a
```

Comment:
Not sure whether I would guard `common.o` behind the individual `USE_DEBUG_xxx` instead of putting it into the common case (maybe later other debug-foo will use it and then the guard becomes almost as long as the other one).

Reply:
Yeah, fair point.  Fixed.